src/interface/components/VLIW/modal/BatchModalComponent.tsx

```diff
@@ -42,8 +42,8 @@ class BatchModalComponent extends React.Component<any, any> {
     setOptions() {
         VLIWIntegration.setBatchMode(
             this.state.replications,
-            this.state.cacheFailPercentage,
-            this.state.cacheFailLatency
+            this.state.cacheFailLatency,
+            this.state.cacheFailPercentage
```

Comment:
Here it is, the importance of [making illegal states unrepresentable](https://ybogomolov.me/making-illegal-states-unrepresentable)! (see "Opaque Types" in this article)